Extractor/extractor.cpp

```diff
@@ -1,37 +1,96 @@
 #include "extractor.h"
+#include "IsoHandler.h"
 #include <argparse.hpp>
 #include <filesystem>
-#include "IsoHandler.h"
 
-void Extractor(const char* iso, const char* output)
+#include "Zero2/Zero2Extractor.h"
+
+void InvalidISOError()
 {
-  auto extractor = IsoHandler::Extractor(iso, output);
+  std::cerr << "Failed to parse ISO file." << std::endl;
 
-  extractor.Extract();
+  std::cerr << "If this is a valid Zero/Fatal Frame/Project Zero ISO file"
```

Comment:
you can use the program arguments ArgumentParser to print a help message instead of coding it yourself